nss-day-cohort-48/rare-client-orchesta-outlaws
# Create/Edit Post (without tags)

#### Changes Made

1. Added `PostForm.js` to `post` directory.

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mec-create-post1
   ```
2. Ensure the backend and client are both running.
3. In the Rare app, click on 'Add Post' to access the new post form
4. Enter input into all fields and click the 'Publish' button
5. The user should be redirected to the My Post view and see the new post published (currently, at the bottom)
6. Click on the edit icon of any of the posts
7. User should be taken to the post form again, this time with the current info about the posts present in the input fields
8. Change any of the input and click on the `Save` button
9. Input changes should be saved and the user redirected to the My Post view
